go/cmd/vtgateclienttest/services/callerid.go

```diff
@@ -77,11 +75,11 @@ func (c *callerIDClient) checkCallerID(ctx context.Context, received string) (bo
 	return true, fmt.Errorf("SUCCESS: callerid matches")
 }
 
-func (c *callerIDClient) Execute(ctx context.Context, session *vtgatepb.Session, sql string, bindVariables map[string]*querypb.BindVariable) (*vtgatepb.Session, *sqltypes.Result, error) {
+func (c *callerIDClient) Execute(ctx context.Context, mysqlCtx vtgateservice.MySQLConnection, session *vtgatepb.Session, sql string, bindVariables map[string]*querypb.BindVariable) (*vtgatepb.Session, *sqltypes.Result, error) {
```

Comment:
I'm thinking we want to introduce a type to be sent in here, instead of growing the arguments across all interface implementations. Like we have the planning-context when planning and the vcursor when running. WDYT?

Reply:
Let's keep that refactor when we go about changing this definition again.